RELEASE_NOTE.md

```diff
@@ -1,6 +1,33 @@
 # OpenPAI Release Note
 
+## Sep 2020 (version 1.2.0)
+
+- Database
+  - New RestServer Arch: RestServer -> DB -> ApiServer (#4651)
+- Webportal
+  - Job list paging in server side
+  - Change job createdTime to submissionTime
+  - VC && Group experience for Admin (AAD Mode) (#4800)
+  - Support SKU count and SKU type in job submission page (#4796)
+  - Upgrade api/v1 code to api/v2 (#4704)
+  - Show "More Diagnostics" in job detail page (#4670)
+- Marketplace
+  - [Unified PAI protocol](https://github.com/microsoft/openpaimarketplace#49)
```

Comment:
can we add issue link as the above for all the items? i.e. #4651, i.e. Job list paging in server side, etc.

Reply:
Added.